lib/querystring.js

```diff
@@ -170,7 +188,11 @@ function stringifyPrimitive(v) {
   return '';
 }
 
-
+/**
+ * @param {string | number | BigInt | boolean} v
```

Comment:
```suggestion
 * @param {string | number | BigInt | boolean} v
```
Same thing as above, type `BigInt` is not recognized, `BigInt` -> `bigint`.